src/main/java/nl/nn/testtool/storage/proofofmigration/ProofOfMigrationStorage.java

```diff
@@ -127,14 +127,15 @@ protected void buildMetadataQuery(int maxNumberOfRecords, List<String> metadataN
 			// The new metadata query will be logged by the super class
 			// E.g.: select min(ID) as min_id, min(TIMESTAMP), COMPONENT, min(concat(CHECKPOINT_NR, '. ', ENDPOINT_NAME)), CORRELATION_ID, count(CHECKPOINT_NR), min(STATUS) from PROOF_OF_MIGRATION where STATUS != 'Success' group by COMPONENT, CORRELATION_ID order by min_id desc limit 10
 		}
-		replace(query, "ID", "min(ID) as min_id");
-		replace(query, "TIMESTAMP", "min(TIMESTAMP)");
-		replace(query, "ENDPOINT_NAME", "min(concat(CHECKPOINT_NR, '. ', ENDPOINT_NAME))");
+		replace(query, "ID", "min(m.ID) as min_id");
+		replace(query, "TIMESTAMP", "i.TIMESTAMP)");
+		replace(query, "CORRELATION_ID", "i.CORRELATION_ID)");
 		replace(query, "NR OF CHECKPOINTS", "count(CHECKPOINT_NR)");
-		replace(query, "STATUS", "min(STATUS)");
-		replace(query, "order by ID" , GROUP_BY + " order by min_id");
+		replace(query, "STATUS", "min(m.STATUS)");
+		replace(query, "MESSAGES", "MESSAGES m, IDS i where m.CORRELATION_ID=i.CORRELATION_ID");
```

Comment:
Can you change it to getTable(), getTable() + " m, IDS i where m.CORRELATION_ID=i.CORRELATION_ID" so it will run with the Ladybug test webapp where I use a different table name?

Can you give me de definition of table IDS to I can add it to https://github.com/ibissource/ibis-ladybug-test-webapp/blob/master/src/main/resources/ProofOfMigrationDatabaseChangelog.xml and test it with the Ladybug test webapp? And maybe also good to share the latest definition of MESSAGES so I can check whether it is in sync with the test webapp?

Reply:
```
	<changeSet id="Create IDS" author="Lenard van der Maas">
		<validCheckSum>8:839af190deab3e852442ca6b6d4e0642</validCheckSum>
		<createTable tableName="IDS">
			<!--Used to correlate all messages from input to output, and requests to responses-->
			<column name="CORRELATION_ID" type="varchar(256)">
				<constraints primaryKey="true" unique="true" nullable="false"/>
			</column>
			<column name="ORIGINAL_ID" type="varchar(256)"/>
			<column name="TIMESTAMP" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP"/>
		</createTable>
	</changeSet>

	<changeSet id="Add timestamp to IDS" author="Gerrit van Brakel" failOnError="false">
		<addColumn tableName="IDS">
			<column name="TIMESTAMP" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP"/>
		</addColumn>
	</changeSet>


	<changeSet id="Add Index to view cases by timestamp, and match to messages" author="Gerrit van Brakel">
		<createIndex tableName="IDS" indexName="IX_IDS_01">
			<column name="TIMESTAMP"/>
			<column name="CORRELATION_ID"/>
		</createIndex>
	</changeSet>
```